core/src/epicli/cli/helpers/objdict_helpers.py

```diff
@@ -27,14 +28,52 @@ def objdict_to_dict(something):
     return _nested_dict_to_dict(something, src_class=dict, dst_class=dict)  # ObjDict is a subclass of dict
 
 
+def is_named_list(l):
+    return all(hasattr(x, 'name') for x in l)
```

Comment:
We can also handle non-iterable objects (such as integers) to avoid errors like `TypeError: 'int' object is not iterable`. The same affects the previous version of this function.

```suggestion
    return isinstance(l, Iterable) and all(hasattr(x, 'name') for x in l)
```

`Iterable` requires importing: `from collections.abc import Iterable`